chart/kubeapps/README.md

```diff
@@ -153,6 +156,87 @@ If you have dedicated a namespace only for Kubeapps you can completely clean the
+  * [Serving Kubeapps in a subpath](#serving-kubeapps-in-a-subpath)
+- [Can Kubeapps install apps into more than one cluster?](#can-kubeapps-install-apps-into-more-than-one-cluster)
+- [Can Kubeapps be installed without Internet connection?](#can-kubeapps-be-installed-without-internet-connection)
+- [Does Kubeapps support private repositories?](#does-kubeapps-support-private-repositories)
+- [Does Kubeapps support Operators?](#does-kubeapps-support-operators)
+- [More questions?](#more-questions)
+
+### How to install Kubeapps for demo purposes?
+
+Install Kubeapps for exclusively **demo purposes** by simply following the [getting started](https://github.com/kubeapps/kubeapps/blob/master/docs/user/getting-started.md) docs.
+
+### How to install Kubeapps in production scenarios?
+
+For any user-facing installation, you should [configure an OAuth2/OIDC provider](https://github.com/kubeapps/kubeapps/blob/master/docs/user/using-an-OIDC-provider.md) to enable secure user authentication with Kubeapps and the cluster.
+Please also refer to the [Access Control](https://github.com/kubeapps/kubeapps/blob/master/docs/user/access-control.md) documentation to configure fine-grained access control for users.
+
+### How to use Kubeapps?
+
+Have a look at the [dashboard documentation](https://github.com/kubeapps/kubeapps/blob/master/docs/user/dashboard.md) for knowing how to use the Kubeapps dashboard: deploying applications, listing and removing the applications running in your cluster and adding new repositories.
+
+### How to configure Kubeapps with Ingress
+
+The example below will match the URL `http://example.com` to the Kubeapps dashboard. For further configuration, please refer to your specific Ingress configuration docs (e.g., [NGINX](https://github.com/kubernetes/ingress-nginx) or [HAProxy](https://github.com/haproxytech/kubernetes-ingress)).
+
+```bash
+helm install kubeapps --namespace kubeapps \
+  --set ingress.enabled=true \
+  --set ingress.hostname=example.com \
+    bitnami/kubeapps
+```
+#### Serving Kubeapps in a subpath
+
+You may want to serve Kubeapps with a subpath, for instance `http://example.com/subpath`, you have to set the proper Ingress configuration. If you are using the ingress configuration provided by the Kubeapps chart, you will have to set the `ingress.extraHosts` parameter:
+
+```bash
+helm install kubeapps --namespace kubeapps \
+    --set ingress.enabled=true
+    --set ingress.extraHosts[0].name="console.example.com"
+    --set ingress.extraHosts[0].path="/catalog"
+    bitnami/kubeapps
+```
+> This will also create an additional ingress rule serving Kubeapps at `kubeapps.local/`. In case you don't want it, you can always patch the ingress object generated before (where we just set the `ingress.hostname`) by executing: `kubectl patch ingress kubeapps -n kubeapps --type=json -p='[{"op": "replace", "path": "/spec/rules/0/http/paths/0/path", "value":"/subpath"}]'`
```

Comment:
You can simply use `--set ingress.hostname=""` to avoid the default hostname.

Reply:
Ah, so the `{{ if .Values.ingress.hostname }}`  will return false if an empty string is passed. Interesting... If so, that's much much better than patching the ingress :facepalm: 
Thanks for the tip!